app/views/projects/show.html.erb

```diff
@@ -1,3 +1,3 @@
 <% if user_signed_in? || organization_signed_in? %>
-    <%= react_component('projects/ProjectView', { project: @project, photos: @photos, project_type: @project_type, organization: @project.organization, org_image_url: @org_image_url, organization_signed_in: organization_signed_in?, current_organization: current_organization, applications: @applications }) %>
+    <%= react_component('projects/ProjectView', { project: @project, photos: @photos, project_type: @project_type, skills: @skills, organization: @project.organization, org_image_url: @org_image_url, organization_signed_in: organization_signed_in?, current_organization: current_organization, applications: @applications }) %>
```

Comment:
I don't think `@skills` is set in the controller, so you will have to change something in the controller to like `@skills = @project.skills`, or just pass in `skills: project.skills`.

Reply:
Projects don't have a skills field (from schema.rb) and I think that you names it skills (taken from projects_controller.rb 
![image](https://user-images.githubusercontent.com/33055010/56870457-78a38680-69c4-11e9-8dfb-34cb90ab6e6d.png)

Reply:
even though projects doesn't have a `skills` field in the schema, rails infers that projects have skills, which is set by the relations defined in the model files. So, in a controller or `.erb` file, u can do `project.skills`, just not in react though unless you do the thing that we did in the `edit` portion of the controller.